.github/workflows/pr.yml

```diff
@@ -17,7 +17,34 @@ jobs:
     needs: lint
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
+      - uses: actions/setup-go@v4
         with:
           go-version: 1.19
-      - run: go test -v ./...
\ No newline at end of file
+      - run: go test -v ./...
+  example-directories:
```

Comment:
Suggestion: should we call this `setup-example-directories` and then possibly rename examples?